src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/trust/CorporateDao.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.dao.trust;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+import uk.gov.companieshouse.overseasentitiesapi.model.dao.AddressDao;
+
+import java.time.LocalDate;
+
+public class CorporateDao {
```

Comment:
could you name the classes to match the definitions in the spec?
CorporateDao -> TrustCorporateDao
IndividualDao -> TrustIndividualDao

CorporateDto -> TrustCorporateDto
HistoricalBoDto -> HistoricalBeneficialOwnerDto
IndividualDto -> TrustIndividualDto